streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamKTableJoinProcessor.java

```diff
@@ -42,24 +42,24 @@
 import static org.apache.kafka.streams.processor.internals.metrics.TaskMetrics.droppedRecordsSensor;
 import static org.apache.kafka.streams.state.ValueAndTimestamp.getValueOrNull;
 
-class KStreamKTableJoinProcessor<K1, K2, V1, V2, VOut> extends ContextualProcessor<K1, V1, K1, VOut> {
+class KStreamKTableJoinProcessor<K, V, KTable, VTable, VOut> extends ContextualProcessor<K, V, K, VOut> {
```

Comment:
Flipping type order -- no splash radius -- change limited to this line only, but diff is weird.

Effectively renaming `K1 -> K`, `V1 -> V`, `K2 -> KTable`, and `V2 -> `VTable`.